packages/block-library/src/navigation/editor.scss

```diff
@@ -212,6 +164,21 @@ $color-control-label-height: 20px;
  * Setup state
  */
 
+// Ensure that an empty block has space around the appender.
+// @todo: revisit this to allow thinner menus.
```

Comment:
The min-height should ideally only be applied to the setup state showing the "skeleton indicators", otherwise we make it harder for themes to create small menus, like this one:

<img width="1157" alt="Screenshot 2021-03-02 at 12 19 46" src="https://user-images.githubusercontent.com/1204802/109641090-99553e00-7b51-11eb-833b-74061497d0ae.png">

But I'd like to do this separately.